### PR TITLE
Added blog posts for every release

### PR DESCRIPTION
**Pull request by sreichel**

### CODE REVIEW

File: _posts/2019-01-20-after-m1-eol.md [hunk 1]
```diff
@@ -93,3 +93,6 @@ OpenMage is competing with solutions like Shopware, Sylius and Shopify.
 OpenMage is for Companies, which for diverse reasons would not switch to Magento 2 anyway, but would consider to switch to one of the other Platforms. For them, OpenMage is the more cost efficient solution, as it only leads to the price of an update, not a replatforming.
 
 
+---
+
+If you want to help us with maintenance, contact us on [Github](https://github.com/OpenMage/magento-lts), [Twitter](https://twitter.com/OpenMageProject) or via [Discord](https://discord.gg/EV8aNbU).
```
Comment (Flyingmana) [inline _posts/2019-01-20-after-m1-eol.md:98]: if we want to have a specific footer on posts, then we should put it into the templates

**Review by Flyingmana — CHANGES_REQUESTED** [review]

File: _posts/2019-01-20-after-m1-eol.md (hunk 1 above)
Comment (sreichel) [reply _posts/2019-01-20-after-m1-eol.md:98]: I have only added links to existing footer.

Feel free to update.

**Comment by fballiano** [comment]
niceeeeee

**Comment by sreichel** [comment]
No, not nice ... this was not done in a few minutes and the only thing is Daniel sees is a footer template that could be added.

Yeah, add some docs how to improve that ..... 

THANK YOU for reviewing this.

**Comment by fballiano** [comment]
> No, not nice ... this was not done in a few minutes and the only thing is Daniel sees is a footer template that could be added.

With "nice" I was talking about your work. Which I'm merge right away.

**Review by fballiano — APPROVED** [review]

**Comment by sreichel** [comment]
> With "nice" I was talking about your work.

I know.

But instead of request a change he could also add a PR for the footer template if he wants to improve that.

**Comment by fballiano** [comment]
> But instead of request a change he could also add a PR for the footer template if he wants to improve that.

I know the frustration

**Comment by Flyingmana** [comment]
because putting the same footer manually to 26 blog posts is a No Go. And from someone who is so much into clean code, I expect to understand that its a terrible Idea to introduce something, which on a possible change requires to change this >20 places again manually.

And all of this is not even related to the actual purpose of this PR

File: _posts/2019-01-20-after-m1-eol.md (hunk 1 above)
Comment (Flyingmana) [reply _posts/2019-01-20-after-m1-eol.md:98]: this is factual wrong. In this file you edited there was no footer.

**Comment by sreichel** [comment]
> because putting the same footer manually to 26 blog posts is a No Go. 

Again - a only used the footer that was already there in most post. I just added links.

> which on a possible change requires to change this >20 places again manually.

Replacing this take less a minute.

> And from someone who is so much into clean code, I expect to understand

Nice. I dont like duplicate code, but i dont spent much time for improving that.

File: _posts/2019-01-20-after-m1-eol.md (hunk 1 above)
Comment (sreichel) [reply _posts/2019-01-20-after-m1-eol.md:98]: - https://www.openmage.org/2022/11/09/release-19-4-19-and-20-0-17.html
- https://www.openmage.org/2022/05/19/release-19-4-16-and-20-0-14.html

**Comment by fballiano** [comment]
but @Flyingmana these 3 posts have the same footer
- https://www.openmage.org/2022/11/09/release-19-4-19-and-20-0-17.html
- https://www.openmage.org/2022/05/19/release-19-4-16-and-20-0-14.html
- https://www.openmage.org/2021/08/27/PHP8-compability-release-19-4-15-and-20.0.13.html

also it seems I can't modify the PR (adding commits)

**Comment by sreichel** [comment]
Branch is already deleted.

**Comment by sreichel** [comment]
I'll repon after @Flyingmana has added a template footer :)

**Comment by sreichel** [comment]
@Flyingmana 

Wie wäre es, wenn du selbst mal den Finger ziehst, statt die Leute zu gängeln?

Du hast es doch nichtmal geschafft, die [Versios-Updates](https://github.com/OpenMage/OpenMage.github.io/pulls?q=is%3Apr+author%3Asreichel+is%3Aclosed) zu mergen - und nun soll ich das Template anpassen? Damit ich den PR nächstes Jahr wieder zumache?

"Frustrierend" is untertrieben ...

**Comment by Flyingmana** [comment]
> @Flyingmana 
> 
> Wie wäre es, wenn du selbst mal den Finger ziehst, statt die Leute zu gängeln?
> 
> Du hast es doch nichtmal geschafft, die [Versions-Updates](https://github.com/OpenMage/OpenMage.github.io/pulls?q=is%3Apr+author%3Asreichel+is%3Aclosed) zu mergen - und nun soll ich das Template anpassen? Damit ich den PR nächstes Jahr wieder zumache?
> 
> "Frustrierend" is untertrieben ...

I like to suggest you to more often check back if a change is actually wanted before you put a lot of work into, this could save a lot of frustration.
Its a repeating pattern that you start doing labor intensive things without checking, and once a year this fires back heavy and you demand to be thankful for something which would have been better solved in a different way.

**Comment by sreichel** [comment]
> Its a repeating pattern that you start doing 

I dont see any problem here. If that "footer" needs change ... search + replace?

I agree, a template file would be better ..... waiting for your PR.

**Comment by sreichel** [comment]
> Stability is an important topic for me. Most times when I blocked a PR, it was falling back to this Goal.
>  ...
>   * it did change multiple things add once (which )
>     * like not only adding a Blog Post, but also changing existing ones
>     * and also changing the formate of something in other ones

Are you _putting formalities over user impact. (because there were no negative downsides on users)_?

- I took the most recent posts as blueprint for new posts. 
- these posts have _"If you want to help us with maintenance, contact us on Github, Twitter or via Discord."_ as last line

My changes to existing posts ...

- adding links to github, twitter, discord
- adding some lines to link github releases
- adding links to mentioned security advisories

Are there any downsides? You blocked it b/c of repeating pattern (links to get in contact) that should move to a template file.

So you want me to read the jekyll docs, how to include files. Whats the right place for this file? Should reusable URLs be added to `_data`? And so one ...

**Comment by Flyingmana** [comment]
or, you remove this footer changes for now, at least from the already existing posts.

And someone opens an issue to discuss if they should be added on all posts, or just release posts, or if there is any exceptions.
Should it have the link or not. Should it be part of the content block, or not. Should it also appear in the feed reader content?

**Comment by fballiano** [comment]
maybe we can remove this "footer" completely? at the end of the day the links are right below, in the real footer.

**Comment by sreichel** [comment]
Not sure about to remove them completly .... most releases had no description (at github), so there is no "content", only links to releases. I'd prefer to keep it for new posts.

**Comment by Flyingmana** [comment]
another note, if it should be for all blog posts, this is the template file
https://github.com/OpenMage/OpenMage.github.io/blob/master/_layouts/post.html#L44-L46

**Comment by sreichel** [comment]
> another note ....

Mach verdammte Axt selbst mal was ...

T ... oll
E ... in
A ... anderer
M ... machts.

Ich bin endgültig raus.

**Pull request closed without merging**